vcsim/README.md

```diff
@@ -20,34 +20,35 @@ to that of the original *vcsim* mode that was included with vCenter.  The number
 of resources can be increased or decreased using the various resource type
 flags.  Resources can also be created and removed using the API.
 
-Example using the default settings:
+Example inventory using the default settings:
 
 ```console
 % export GOVC_URL=https://user:pass@127.0.0.1:8989
 % $GOPATH/bin/vcsim
-% govc find
-/
-/DC0
-/DC0/vm
-/DC0/vm/DC0_H0_VM0
-/DC0/vm/DC0_H0_VM1
-/DC0/vm/DC0_C0_RP0_VM0
-/DC0/vm/DC0_C0_RP0_VM1
-/DC0/host
-/DC0/host/DC0_H0
-/DC0/host/DC0_H0/DC0_H0
-/DC0/host/DC0_H0/Resources
-/DC0/host/DC0_C0
-/DC0/host/DC0_C0/DC0_C0_H0
-/DC0/host/DC0_C0/DC0_C0_H1
-/DC0/host/DC0_C0/DC0_C0_H2
-/DC0/host/DC0_C0/Resources
-/DC0/datastore
-/DC0/datastore/LocalDS_0
-/DC0/network
-/DC0/network/VM Network
-/DC0/network/DVS0
-/DC0/network/DC0_DVPG0
+% govc find -l
+Folder                       /
+Datacenter                   /DC0
+Folder                       /DC0/vm
+VirtualMachine               /DC0/vm/DC0_H0_VM0
+VirtualMachine               /DC0/vm/DC0_H0_VM1
+VirtualMachine               /DC0/vm/DC0_C0_RP0_VM0
+VirtualMachine               /DC0/vm/DC0_C0_RP0_VM1
+Folder                       /DC0/host
+ComputeResource              /DC0/host/DC0_H0
+HostSystem                   /DC0/host/DC0_H0/DC0_H0
+ResourcePool                 /DC0/host/DC0_H0/Resources
+ClusterComputeResource       /DC0/host/DC0_C0
+HostSystem                   /DC0/host/DC0_C0/DC0_C0_H0
+HostSystem                   /DC0/host/DC0_C0/DC0_C0_H1
+HostSystem                   /DC0/host/DC0_C0/DC0_C0_H2
+ResourcePool                 /DC0/host/DC0_C0/Resources
+Folder                       /DC0/datastore
+Datastore                    /DC0/datastore/LocalDS_0
+Folder                       /DC0/network
+Network                      /DC0/network/VM Network
```

Comment:
Should we add `OpaqueNetwork`?

Reply:
Good idea, I've expanded an Examples section that includes this and more. Can you review again?